plugins/node/opentelemetry-instrumentation-express/.tav.yml

```diff
@@ -0,0 +1,15 @@
+express:
+  - versions: "^4.17.2"
+    commands: npm run test
+
+  - versions: "4.17.1"
```

Comment:
Any reason you didn't use `^4.16.2` for the range which matches all the listed versions?

Reply:
I too think it's more appropriate here